docs/cli/linear.rst

```diff
@@ -9,19 +70,32 @@ For training, use
 .. code-block:: bash
 
     python3 main.py --config CONFIG_PATH \
-                    --linear [--liblinear_options LIBLINEAR_OPTIONS]
+                    --linear \
+                    --liblinear_options LIBLINEAR_OPTIONS \
+                    --linear_technique MULTILABEL_TECHNIQUE
 
-- **config**: configure parameters in a yaml file.
+- **config**: configure parameters in a yaml file. Notice that the validation set is not needed to be specified additionally as the program will ignore it.
```

Comment:
```suggestion
- **config**: configure parameters in a yaml file.  A validation set is not needed because the program may split the training set for internal validation. If specified, it will be ignored.
```